### PR TITLE
Closes #1507: RDS citations

### PR DESCRIPTION
**Pull request by wtr-icm**
Probably there will be small changes to some citations – but rather non-substantial.

### CODE REVIEW

File: src/main/java/edu/harvard/iq/dataverse/citation/RdsCitationFormatsConverter.java [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+        if (data.getFileTitle() != null && data.isDirect()) {
+            citation.add("\"").value(data.getFileTitle()).endPart("\", ")
+                    .add("<i>").value(data.getTitle()).add("</i>");
+        } else {
+            citation.add("\"").value(data.getTitle()).add("\"");
+        }
+        citation.add(getConstant(Constants.DATA)).endPart(". ");
+
+        if (!data.getProducers().isEmpty()) {
+            citation.add(joinProducers(data)).endPart(StringUtils.EMPTY)
```
Comment (madryk) [inline src/main/java/edu/harvard/iq/dataverse/citation/RdsCitationFormatsConverter.java]: `citation.add(joinProducers(data))` - seems unsafe if we produce html citation. We can't tell what user will enter in those dataset fields

File: src/main/java/edu/harvard/iq/dataverse/citation/RdsCitationFormatsConverter.java [hunk 2]
```diff
@@ -0,0 +1,271 @@
+package edu.harvard.iq.dataverse.citation;
```
Comment (madryk) [inline src/main/java/edu/harvard/iq/dataverse/citation/RdsCitationFormatsConverter.java]: Can we make a package that reflects groupId and artifactId of this maven project (`pl.edu.icm.rds.extension.citation`)?

**Review by madryk — CHANGES_REQUESTED** [review]

File: src/main/java/edu/harvard/iq/dataverse/citation/RdsCitationFormatsConverter.java (hunk 2 above)
Comment (wtr-icm) [reply src/main/java/edu/harvard/iq/dataverse/citation/RdsCitationFormatsConverter.java]: Theoretically yes, but in fact I've made some of the classes or methods in main repository `package private`, so in that case I'd have to make some of these `public` (or `protected`).

Comment (madryk) [reply src/main/java/edu/harvard/iq/dataverse/citation/RdsCitationFormatsConverter.java]: I would go with `protected` as it is intended to use them by implementing classes

**Review by madryk — APPROVED** [review]

**Pull request merged**